zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -291,7 +314,10 @@ public Interpreter getCurrentRepl() {
 
     String replName = getRequiredReplName(buffer);
     if (replName != null && cursor > replName.length()) {
-      cursor -= replName.length() + 1;
+      Matcher m = Pattern.compile(String.format("%%%s\\s*", replName)).matcher(buffer);
```

Comment:
@felixcheung 
L307 logic to find replName. Now problem with whitespace character. For example
```
%replName\n
\n
\n
text<cursor>
```
result **cursor=7** but method **getScriptBody** return ```text```

Need count all characters (for correct value of cursor).